app/src/main/java/org/wikipedia/analytics/eventplatform/EventPlatformClient.java

```diff
@@ -169,9 +171,14 @@ static void addEventMetadata(Event event) {
         private static final int WAIT_MS = 30000;
 
         /*
-         * When QUEUE.size() exceeds this value TIMER becomes non-interruptable.
+         * While coming out of offline, When QUEUE.size() exceeds this value TIMER becomes non-interruptable.
+         */
+        private static final int MAX_OFFLINE_QUEUE_SIZE = 128;
+
+        /*
+         * While online, When QUEUE.size() exceeds this value TIMER becomes non-interruptable.
          */
-        private static final int MAX_QUEUE_SIZE = 128;
+        private static final int MAX_ONLINE_QUEUE_SIZE = 10;
```

Comment:
One of the design goals of the Event Platform Client libraries is to make network requests for event logging less frequent.  Limiting the queue size to 10 items when online seems at cross-purposes with that goal.  Does batched event submission resolve the concern behind this?

Reply:
You are right @mdholloway , With batching, this can just go back to 128. I was curious... why did we choose 128 as the limit. The upper limit to the POST body itself is 10MB, so in theory this could be much higher?

Reply:
Reverted to 128 for now